app/views/layouts/application.html.erb

```diff
@@ -2,15 +2,36 @@
 <html>
 <head>
   <title>JobTracker</title>
+  <link href="https://fonts.googleapis.com/css?family=Oleo+Script|Josefin+Sans" rel="stylesheet">
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
 </head>
 <body>
+  <nav class="navbar navbar-inverse">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-text" style="color:white;" href="/dashboard"> dashboard </a>
+        <a class="navbar-text" style="color:white;" href="/companies"> companies </a>
+        <a class="navbar-text" style="color:white;" href="/tags"> tags </a>
+        <a class="navbar-text" style="color:white;" href="/categories"> categories </a>
+      </div>
+      <div class="navbar-right login_link">
+        <% if session[:current_user_id] %>
+          <%= "#{User.find(session[:current_user_id]).name} is logged in" %> - 
```

Comment:
again, ideally we don't need to access our models or query our database in our views. how can we make this a helper method?